gr-fec/lib/polar_decoder_common.cc

```diff
@@ -16,8 +16,7 @@
 #include <gnuradio/io_signature.h>
 #include <volk/volk.h>
 
-#include <cstdio>
-#include <iostream>
+#incldue < spdlog / fmt / fmt.h>
```

Comment:
This doesn't compile, for both wrong spaces and because you misspelled `#include`. Don't submit C++ PRs that you can't build locally!

Reply:
Sorry again, made the changes please check